.coveragerc

```diff
@@ -143,6 +143,9 @@ omit =
     homeassistant/components/netatmo.py
     homeassistant/components/*/netatmo.py
 
+    homeassistant/components/nuheat.py
+    homeassistant/components/*/nuheat.py
```

Comment:
should this be omitted?

Reply:
Since you have written test cases by mocking the thermostat ( :+1: ), I think this file should count towards the coverage, i.e., not be listed here.

Reply:
awesome. will remove. the instructions for opening new PRs confused me on this one.